certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -26,8 +26,8 @@
 from certbot_integration_tests.utils import misc
 
 
-@pytest.fixture()
-def context(request):
+@pytest.fixture(name='context')
+def test_context(request):
```

Comment:
What is the error triggered by pylint behind this modification?

Reply:
Masking context from outer scope.

Reply:
`W0621: Redefining name 'context' from outer scope (line 30) (redefined-outer-name)`